src/vs/workbench/contrib/terminalContrib/suggest/common/terminalSuggestConfiguration.ts

```diff
@@ -65,7 +65,7 @@ export const terminalSuggestConfiguration: IStringDictionary<IConfigurationPrope
 	},
 	[TerminalSuggestSettingId.Providers]: {
 		restricted: true,
-		markdownDescription: localize('suggest.providers', "Controls which providers are enabled for terminal suggestions."),
+		markdownDescription: localize('suggest.providers', "Controls which providers are enabled for terminal suggestions. A provider is enabled by default."),
```

Comment:
Should this be "_All_ providers are enabled by default"? You could just omit this sentence and instead explain what a provider is (the id of a provider from an extension?). It'll be simpler when  we get rid of `pwsh-shell-integration` which will probably be next month

Reply:
I tweaked the language. I wan tto leave `terminal-suggest` `true` for now bc we don't have intellisense that provides those yet.